src/nls/hr/strings.js

```diff
@@ -62,7 +62,7 @@ define({
     "ERROR_CREATING_FILE"               : "Došlo je do greške prilikom pokušaja da se kreira {0} <span class='dialog-filename'>{1}</span>. {2}",
 
     // Application preferences corrupt error strings
-    "ERROR_PREFS_CORRUPT_TITLE"         : "Greška kod čitanja preferencija",
+    "ERROR_PREFS_CORRUPT_TITLE"         : "Greška prilikom učitanja preferencija",
```

Comment:
I think the fork got destroyed at some point, but it's not a big deal. We can merge this and fix the one type in the next PR.